src/docs/product/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -5,11 +5,11 @@ redirect_from:
 description: "Learn more about Sentry's Integration Platform User Interface (UI) components and how they provide developers with the ability to define UI components to render within specific parts of Sentry's UI, through a JSON-based schema."
 ---
 
-The Sentry Integration Platform provides developers with the ability to define User Interface components to render within specific parts of Sentry's UI, through a JSON-based schema. Two possible use-cases are [Issue Link](#issue-link) and [Stack Trace Link](#stack-trace-link).
+The Sentry Integration Platform provides developers with the ability to define User Interface components to render within specific parts of Sentry's UI, through a JSON-based schema. The possible use-cases are [Issue Link](/product/integrations/integration-platform/ui-components/issue-link), [Stack Trace Link](/product/integrations/integration-platform/ui-components/stacktrace-link), and [Alert Rule Actions](/product/integrations/integration-platform/ui-components/alert-rule-action). Certain UI components can build forms using [FormField](/product/integrations/integration-platform/ui-components/formfield) components.
 
-# Schema
+## Schema
 
-The UI components are specified in the schema section of integration details. The required form of this schema is:
+The UI components are specified in the schema section of the integration details. The required form of this schema is:
 
 ```javascript
```

Comment:
nit: 'json' and {filename: schema.json}

Reply:
What do you mean by {filename: schema.json}?